docs/governance/community-guidelines/releases.md

```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the CCC project.
+
+## Ownership
+
+The [Communications WG] takes ownership of releases. They set the roadmap and communicate project progress to the wider community. All other [WG]s should contribute to determining the content of each release.
+
+## Scheduling
+
+The community aims for several releases per year. Releases can occur as soon as the controls for a service are complete and approved. The CCC project should prioritize stable releases and where possible avoid excessive release frequency (e.g., multiple times per day).
```

Comment:
There's quite a range between "several a year" vs "multiple per day". More an observation than anything else

Reply:
What constitutes stable in this context?

Reply:
not multiple releases per day :)